src/components/layout/layout.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const Layout: React.FC<Props> = ({ children }: Props) => {
```

Comment:
`({ children }: Props)` can be just `({ children })` since the type is inferred from `React.FC<Props>`

Reply:
Alright